frontend/src/profiles/uvp/reports/activity-report/activity-report.component.ts

```diff
@@ -151,6 +151,7 @@ export class ActivityReportComponent implements AfterViewInit {
             contact_uuid: entry.contact_uuid,
             actor: entry.actor,
             action: this.translateAction(entry.action),
+            state: entry.action,
```

Comment:
`deleted: entry.action == 'delete',`

Refactor this so the logic is kept out of the template.